ccm_web/server/src/lti/lti.service.ts

```diff
@@ -45,8 +46,32 @@ export class LTIService implements BeforeApplicationShutdown {
 
     // Redirect to the application root after a successful launch
     provider.onConnect(async (token: IdToken, req: Request, res: Response) => {
-      logger.debug(JSON.stringify(token.userInfo))
-      return provider.redirect(res, '/')
+      logger.debug(`The LTI launch was successful! User info: ${JSON.stringify(token.userInfo)}`)
+      const customLTIVariables = token.platformContext.custom
+      if (customLTIVariables?.login_id === undefined) {
```

Comment:
If we don't have any custom properties defined in LTI Developer key the `const customLTIVariables = {}`

![Screen Shot 2021-05-24 at 2 16 36 PM](https://user-images.githubusercontent.com/8579775/119392643-0504fe00-bc9e-11eb-8de5-320f5e3fbb63.png)

So this check is still valid. If we miss adding `login_id` in the custom variable pool, that it is undefined. So this check I am going is still valid.

![Screen Shot 2021-05-24 at 2 31 01 PM](https://user-images.githubusercontent.com/8579775/119392661-0a624880-bc9e-11eb-8245-5b91ba23b36f.png)

I think my check is still valid

Reply:
The check is valid (I think I suggested it earlier), but I think you can just do `customLTIVariables.login_id === undefined` (instead of `customLTIVariables?.login_id === undefined`).

Reply:
Done